Tanvir04Khan/01-max-udemy/03-login-page/src/Components/Home/HomePage.js

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment } from "react";
+
+import HomeHeader from "./HomeHeader";
+import Card from "../UI/Card";
+
+import "./HomePage.css";
+
+const HomePage = ({ onAuthenticate }) => {
+  const handelLogout = () => {
+    onAuthenticate(false);
+    localStorage.removeItem("isLoggedIn");
+  };
+  return (
+    <Fragment>
+      <HomeHeader onLoggingout={handelLogout} />
```

Comment:
This should be onLogout

Reply:
changed code as per suggestion @faran4engg